src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
changing to static should be in T4A2. Please create another branch to do it, and create another pull request for that exercise. :)